aws_xray_sdk/core/emitters/udp_emitter.py

```diff
@@ -62,7 +65,7 @@ def _send_data(self, data):
             self._socket.sendto(data.encode('utf-8'), (self._ip,
                                 self._port))
         except Exception:
-            log.exception('failed to send data to X-Ray daemon.')
+            raise
```

Comment:
Do we need to explicitly `raise` the exception here if we are not specially handling it? I think we can remove the `try-except` block here.